.github/scripts/ci_tests.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
```

Comment:
Nit - is .github the directory stuff something like this should live?  I've normally seen this for bread&butter hub setup and the tests live "someplace else."  If this is a convention I'm not aware of it's fine as is, but want to make sure we're in alignment.

Reply:
all the github Actions get defined in this folder (.github/workflow/). So all the scripts that are getting used in this workflow are kept under it only.